requirements.txt

```diff
@@ -16,21 +16,17 @@ certbot-dns-ovh==2.6.0
 cloudflare==2.11.6
 container-runtime-interface-api==1.0.0
 dns-lexicon==3.12.0
-google-api==0.1.12
 google-api-core==2.11.1
 google-api-python-client==2.93.0
 googleapis-common-protos==1.59.1
 google-auth==2.22.0
 google-auth-httplib2==0.1.0
-ipaddr==2.2.0
 jsonlines==3.1.0
 pyRFC3339==1.1
 git+https://github.com/truenas/python-prompt-toolkit.git@84d42cd#egg=prompt-toolkit==3.0.37
 protobuf==4.24.4
 pyroute2==0.7.9
 python-digitalocean==1.17.0
-PyYAML==6.0.1
 remote-pdb==2.1.0
 s3transfer==0.6.1
-tableprint==0.9.1
```

Comment:
We may actually need this for midcli.

Reply:
Actually, we don't, midcli control file is at fault https://github.com/truenas/midcli/pull/109